spring-amqp/src/main/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverter.java

```diff
@@ -53,23 +53,36 @@
 	private boolean typeMapperSet;
 
 	/**
+	 * Construct with an internal {@link ObjectMapper} instance and trusted packed to all ({@code *}).
+	 * @since 1.6.11
+	 */
+	public Jackson2JsonMessageConverter() {
+		this("*");
+	}
```

Comment:
@artembilan I think we need this on the default class mapper (at least for backports). Agree?